### PR TITLE
Fixed "Crying over Onions" blocking Nanaa Mihgo

### LINKED ISSUE #880: Nanaa Mihgo: Quest "Crying Over Onions" blocks Thief AF1 "Tenshodo Showdown"
**_I have:_**

- [x] searched existing issues (http://project-topaz.com/issues/) to see if the issue has already been opened
- [x] checked the commit log to see if the issue has been resolved since my server was last updated

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 
Steps to Reproduce: 
- Quest Crying Over Onions Flagged
- Thief Level 40
- Speaking with NPC any number of times, will only reference Crying Over Onions quest

Expected Behavior
- First dialogue: Related to Crying Over Onions quest
- Second dialog: Related to Tenshodo Showdown quest

OR

- First Dialogue: Related to Tenshodo Showdown quest
- Second Dialogue: Related to Crying Over Onions quest

Should be some temporary flag to allow next dialogue to execute that is cleared on zone that will mark Mana orb dialogue as read. Alternatively, moving the elseif statement for Tenshodo Showdown before Crying Over Onions should resolve.

### PR DESCRIPTION
**Pull request by MarianArlt**
**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

Nanaa Mihgo now increases the charVar bit responsible for this to not make her repeat the dialogue regarding this quest. The remaining NPCs involved in "Crying for Onions" got their bit increased accordingly. Tested working locally and extensively by talking to every NPC before the quest, in every step during the quest, and after the quest.
Closes #880

### CODE REVIEW

File: scripts/zones/Windurst_Waters/npcs/Honoi-Gomoi.lua [hunk 1]
```diff
@@ -68,12 +79,17 @@ function onEventFinish(player, csid, option)
...
+        player:setCharVar("CryingOverOnions", 3)
+    elseif csid == 776 and npcUtil.completeQuest(player, WINDURST, tpz.quest.id.windurst.CRYING_OVER_ONIONS, {
+        fame=120,
+        var="CryingOverOnions"
+    }) then
         player:needToZone(true)
     elseif csid == 780 then
         player:addQuest(WINDURST, tpz.quest.id.windurst.WILD_CARD)
-    elseif csid == 782 and npcUtil.completeQuest(player, WINDURST, tpz.quest.id.windurst.WILD_CARD, {title=tpz.title.DREAM_DWELLER, fame=135, var="WildCard"}) then
+    elseif csid == 782 and npcUtil.completeQuest(player, WINDURST, tpz.quest.id.windurst.WILD_CARD, {
```
Comment (TeoTwawki) [inline scripts/zones/Windurst_Waters/npcs/Honoi-Gomoi.lua]: when we make a conditional span multi line, I think its easier to read when we put a newline at both ends of the condition and align the `then` to the start:

```lua
    elseif
        csid == 782 and 
        npcUtil.completeQuest(player, WINDURST, tpz.quest.id.windurst.WILD_CARD, {
            title=tpz.title.DREAM_DWELLER, fame=135, var="WildCard"
        })
    then
```

Comment (MarianArlt) [reply scripts/zones/Windurst_Waters/npcs/Honoi-Gomoi.lua:90]: Options of `npcUtil.completeQuest()` on one line or listed?

Comment (MarianArlt) [reply scripts/zones/Windurst_Waters/npcs/Honoi-Gomoi.lua]: Changed in accordance with **Inline tables** section from [Style Guide](https://github.com/project-topaz/topaz/blob/release/CONTRIBUTING.md).

**Comment by MarianArlt** [comment]
Trying to git amend on behalf of the Style Guide suggestions left me with a broken merge conflict which I am unable to resolve,...sigh. Line 93-94 should be below the comment `"-- Wild Card"`

File: scripts/zones/Windurst_Waters/npcs/Honoi-Gomoi.lua (hunk 1 above)
Comment (TeoTwawki) [reply scripts/zones/Windurst_Waters/npcs/Honoi-Gomoi.lua]: break at curly brace is what the existing examples and guide did/do

**Comment by TeoTwawki** [comment]
> Trying to git amend on behalf of the Style Guide suggestions left me with a broken merge conflict which I am unable to resolve,...sigh. Line 93-94 should be below the comment `"-- Wild Card"`

did you happen to try and pull after amending? coz you don't want to do that if git push errors, the error would likely be from failing to do a force push instead of a regular push.

**Comment by zach2good** [comment]
Looks good to me, thanks for all the cleanup too 👍

**Pull request merged**